### PR TITLE
REFACTOR-#5607: Fix missing formatting with 'black'

### LINKED ISSUE #5607: Align missing formatting with 'black'
'black' has released a new version where it seems to now spot some new cases of deviation from a code style. We have to align accordingly to unblock our CI

### PR DESCRIPTION
**Pull request by dchigarev**
## What do these changes do?

- [x] first commit message and PR title follow format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
  > **_NOTE:_**  If you edit the PR title to match this format, you need to add another commit (even if it's empty) or amend your last commit for the CI job that checks the PR title to pick up the new PR title.
- [x] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x] Resolves #5607  
- [ ] tests added and passing
- [x] module layout described at `docs/development/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by anmyachev** [comment]
@dchigarev if these changes are not compatible with previous versions of `black`, then we must update our environments by adding `black` minimum version.

**Comment by dchigarev** [comment]
> @dchigarev if these changes are not compatible with previous versions of `black`, then we must update our environments by adding `black` minimum version.

All of the changes are actually backward compatible, meaning that the old 'black' won't complain about this new formatting. The only lines that cause problems are [these two at `test_series.py`](https://github.com/modin-project/modin/pull/5606/files#diff-c4220d25a8d9829abdcb6691e2e69f78eeeb4ae7c205987b246199514b40156aR2210). I've marked them as `# fmt: skip` as the latest black + latest flake8 can't agree on how to format this place.

File: modin/pandas/test/test_series.py [hunk 1]
```diff
@@ -2204,12 +2204,13 @@ def test_loc(data):
...
     pandas_series = pandas.Series(data, index=index).sort_index()
+    # Using 'fmt: skip' below as 'black' and 'flake8' can't agree on how this should be formatted
     modin_result = modin_series.loc[
         (slice(None), 1),
-    ]
+    ]  # fmt: skip
     pandas_result = pandas_series.loc[
         (slice(None), 1),
-    ]
+    ]  # fmt: skip
```
Comment (dchigarev) [inline modin/pandas/test/test_series.py:2213]: `black` wants this to be formatted exactly as:
```python
modin_results = modin_series.loc[(slice(None), 1),]
```
when `flake8` agrees on either of these:
```python
modin_results = modin_series.loc[(slice(None), 1), ]
# or
modin_results = modin_series.loc[
    (slice(None), 1),
]
```
so let's just keep the old formatting

**Review by anmyachev — APPROVED** [review]

File: modin/pandas/test/test_series.py (hunk 1 above)
Comment (dchigarev) [reply modin/pandas/test/test_series.py:2213]: and yes, the comma is mandatory here as it decides whether the `.__getitem__` receives a single tuple or a nested one:
```python
>>> pandas_series.loc[(slice(None), 1)]
f
0     1
1    11
...
>>> pandas_series.loc[(slice(None), 1),]
f  s
0  1     1
1  1    11
...
```

**Comment by dchigarev** [comment]
@anmyachev does `test_io` failure should be fixed in #5603?

**Comment by anmyachev** [comment]
> @anmyachev does `test_io` failure should be fixed by #5603?

I believe so

**Comment by dchigarev** [comment]
> > @anmyachev does `test_io` failure should be fixed by #5603?
> 
> I believe so

Well, we can't get green CI on these PRs separately then... One of them has to be merged with failing CI, unfortunately.

@modin-project/modin-core, opinions, what should we do?

**Comment by anmyachev** [comment]
> > > @anmyachev does `test_io` failure should be fixed by #5603?
> > 
> > 
> > I believe so
> 
> Well, we can't get green CI on these PRs separately then... One of them has to be merged with failing CI, unfortunately.
> 
> @modin-project/modin-core, opinions, what should we do?

It's just a code style change, it's very unlikely that something will break, let's just merge it and fix another problem.

**Review by mvashishtha — APPROVED** [review]

**Pull request merged**